src/server/api/common/generate-replies-query.ts

```diff
@@ -1,22 +1,24 @@
 import { User } from '@/models/entities/user';
 import { Brackets, SelectQueryBuilder } from 'typeorm';
 
-export function generateRepliesQuery(q: SelectQueryBuilder<any>, me?: { id: User['id'] } | null) {
-	if (me == null) {
+export function generateRepliesQuery(q: SelectQueryBuilder<any>, me: any) {
```

Comment:
you shouldn't use TypeScript's `any` in normal case.

```suggestion
export function generateRepliesQuery(q: SelectQueryBuilder<any>, me?: Pick<User, 'id' | 'showTimelineReplies'> | null) {
```

Reply:
Yeah, sorry about that :sweat_smile: I assumed this wasn't good but TypeScript isn't a language I work with a lot. Thank you!